css/culture.css

```diff
@@ -35,24 +35,70 @@ body {
   display: flex;
   justify-content: center;
   align-items: center;
+  flex-direction: column;
   height: 100%;
   width: 100%;
 }
 
+#counter {
+  color: #f4f4f4;
+  font-family: 'Roboto', sans-serif;
+  margin-top: 20px;
+
+}
+
 #story-text {
   font-size: 28px;
-  width: 900px;
+  max-width: 900px;
   color: #f4f4f4;
   text-shadow: 2px 2px #000000;
   line-height: 1.7em;
-
+  min-height: 250px;
   font-family: 'Roboto', sans-serif;
-
+  border-radius: 10px;
+  padding: 5px 25px;
   opacity: 0;
+  text-align: center;
 }
 
 .done #story-text {
   opacity: 1;
   transition: opacity 1s;
   transition-delay: 0.5s;
+}
+
+#story-text.loaded {
+  transition: opacity 0.05s;
+  transition-delay: 0s;
+}
+
+#story-text.start {
+  opacity: 0;
+}
+
+
+.shake {
+  animation: shake 0.2s cubic-bezier(.36,.07,.19,.97) both;
+  animation-iteration-count: infinite;
+  /* transform: translate3d(0, 0, 0); */
+  /* backface-visibility: hidden; */
+  /* perspective: 1000px; */
+}
+
+@keyframes shake {
```

Comment:
When I tried it I thought this is a bug and now I see you made it a feature. 🙈

Reply:
My UI philosophy here is that every action by the user should give some kind of acknowledgement that it was received. In this case the action the user wants to perform (advancing a slide) is invalid, but I still want to show that we received input. Maybe a shake is not in line with the zen of this page, but it's something

Reply:
That makes totally sense and I am wondering if my brain wasn't on. I thought it is broken because it did move something in the direction I wanted to go but it did not move what I wanted.